src/utilities/stringToCmdLine.cpp

```diff
@@ -16,11 +16,6 @@
 
 namespace utilities
 {
-StringToCmdLine::StringToCmdLine(const std::string &cmdString) : argCount(0)
```

Comment:
Think you misunderstood me here.  this function should be in the C++ file.  the initializer for argCount(0), should be removed and replaced with int argCount=0;  in the class definition.

Reply:
You typically don't want to do the initializers with a constant unless there is a need to do it for overloaded constructors

Reply:
What changes with overloaded constructors?

Specifically, in this PR I'm using `type()` instead of `type =` in src/utilities/saturation.cpp, should that change?